### PR TITLE
HW9 Array-methods task

### PR DESCRIPTION
**Pull request by andreamamchur**

### CODE REVIEW

File: Exercises/arrays-methods/arrays-methods.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// Create a function - predicate which will return `true` if passed element present in array and `false` - if not.
+
+function includes(array, element) {
+  return array.includes(element);
+}
+
+// Create a function which will check that all array elements are even.
+
+function checkEven(array) {
+  return array.every((element) => element % 2 === 0);
```
Comment (lizarusi) [inline Exercises/arrays-methods/arrays-methods.js]: you can omit comparing with `0`

File: Exercises/arrays-methods/arrays-methods.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+// Create a function - predicate which will return `true` if passed element present in array and `false` - if not.
+
+function includes(array, element) {
+  return array.includes(element);
```
Comment (lizarusi) [inline Exercises/arrays-methods/arrays-methods.js]: let's not use includes, as it is too easy :)

**Review by lizarusi — CHANGES_REQUESTED** [review]

File: Exercises/arrays-methods/arrays-methods.js (hunk 1 above)
Comment (andreamamchur) [reply Exercises/arrays-methods/arrays-methods.js]: @lizarusi , I dont have any idea how to check that number is even/odd without comparing the division remainder with 0 :(

File: Exercises/arrays-methods/arrays-methods.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+// Create a function - predicate which will return `true` if passed element present in array and `false` - if not.
+
+// function includes(array, element) {
+//   return array.includes(element);
+// }
+
+function includes(array, element) {
+  return array.some((item) => item / element === 1);
```
Comment (Mike-Savin) [inline Exercises/arrays-methods/arrays-methods.js]: And what if you will try this method for non numeric array?

File: Exercises/arrays-methods/arrays-methods.js (hunk 1 above)
Comment (Mike-Savin) [reply Exercises/arrays-methods/arrays-methods.js]: The result of `% 2` expression is 0 or 1 so it could be converted to boolean without comparion

**Review by Mike-Savin — CHANGES_REQUESTED** [review]

**Review by Mike-Savin — APPROVED** [review]

**Pull request merged**